### PR TITLE
Add test user for tthyer

### PR DESCRIPTION
**Pull request by tthyer**
Add user for Tess to test service catalog

### CODE REVIEW

File: templates/sc-accounts.yaml [hunk 1]
```diff
@@ -22,3 +22,15 @@ Resources:
       LoginProfile:
         Password: !Ref InitNewUserPassword
         PasswordResetRequired: true
+  TthyerTestUser:
+    Type: 'AWS::IAM::User'
+    Properties:
+      UserName: tthyer.test@sagebase.org
```
Comment (zaro0508) [inline templates/sc-accounts.yaml]: this needs to be unique and it doesn't need to be a valid email

Comment (tthyer) [reply templates/sc-accounts.yaml]: Is there something you want me to change? Doesn't this value qualify?

Comment (zaro0508) [reply templates/sc-accounts.yaml]: oops, yes that works.

Comment (zaro0508) [reply templates/sc-accounts.yaml]: oops, yes that works

**Pull request merged**